.flake8

```diff
@@ -15,18 +15,20 @@ ignore =
     W504,      # line break before binary operator (again, I guess).
     S101,      # Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
 
-# F401:  unused import.
-# F403:  cannot detect unused vars if we use starred import
+# F401: unused import.
+# F403: cannot detect unused vars if we use starred import
 # FS003: f-string missing prefix.
 # R102: unnecessary parenthesis on raised exception (raises false positives in places)
+# S106: posible hardcoded password (we dont care about this in the tests)
+# D104, D103, D101, D100: Missing docstring in public function/class/module (ignore in tests)
+# PT011: Set the match parameter for `pytest.raises`
```

Comment:
What is this?

Reply:
There is a parameter you can pass to `pytest.warn` that allows you to make sure the error that was raised is the expected error. We don't use it so i just ignored it. If you want I can try to implement this

Reply:
go for it